python/ray/worker.py

```diff
@@ -1250,6 +1250,22 @@ def get_address_info_from_redis(redis_address,
         counter += 1
 
 
+def setup_logger(logging_level, logging_format):
```

Comment:
`worker.py` is probably not the right file for this. Maybe `utils.py`?

Reply:
done